src/LaurentSeries.jl

```diff
@@ -24,9 +24,11 @@ object `S` will be cached so that supplying the same base ring, string and
 precision in future will return the same parent object and generator. If
 caching of the parent object is not required, `cached` can be set to `false`.
 """
+
+@doc _laurent_series_ring_doc
```

Comment:
Why do we need to attach this docstring to two methods?

Reply:
Well, I can of course also let one refer to the other. I don’t care. Currently, the docstring is only attached to `laurent_series_ring`. Attaching it two both seemed to be an improvement.

Reply:
alright